src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -270,6 +270,11 @@ private void checkClosed() throws SQLServerException {
 
     private static final String IS_GENERATEDCOLUMN = "IS_GENERATEDCOLUMN";
     private static final String IS_AUTOINCREMENT = "IS_AUTOINCREMENT";
+    private static final String SELECT = "SELECT";
+    private static final String WHERE = " WHERE ";
+    private static final String WHERE_CLAUSE = " WHERE 0 = 1";
```

Comment:
This is not a helpful name, when reading where it's used. E.g: 
![image](https://user-images.githubusercontent.com/25423471/202565227-ce300f68-dd65-4e9a-9f16-94d9d525c8c7.png)
Can this be renamed WHERE_ZERO_IS_ONE